### ISSUE TITLE
Important Message

### ISSUE THREAD

**Issue by Isayso** [Isayso/PlaylistEditorTV#72]
Github forces everybody to use 2FA. 
I will probably not enable this. I'm just a hobby programmer and this data grab to my mobile phone is not worth it. 
I highly doubt that somebody will alter my code secretly. 

Over the years all the donations combined were below 50$, so it is not worth the effort anyway.

**Comment by ShareASmile** [comment]
So what are the plans if you have decided! migrate to codeberg etc?

**Comment by Isayso** [comment]
I will probably move to Gitlab, I didn't know codeberg, will take a look at, thanks!

**Comment by ShareASmile** [comment]
PipePipe (formerly AnimePipe) is also developed on codeberg 
https://codeberg.org/NullPointerException/PipePipe

NewPipe code backups were also taken on codeberg in the past in fear of DMCA takedown, but i haven't checked recently.

**Comment by Isayso** [comment]
Thanks for the tip for codeberg, opened an account, same name, already migrated some code. Even the connection to Android Studio simply worked. Had lots of problems to push code for Newpipe19 to Github this year. 

I like BraveNewpipe with Rumble and Bitchute access.

**Comment by slippyC** [comment]
> Github forces everybody to use 2FA. I will probably not enable this. I'm just a hobby programmer and this data grab to my mobile phone is not worth it. I highly doubt that somebody will alter my code secretly.
> 
> Over the years all the donations combined were below 50$, so it is not worth the effort anyway.

I agree, 2FA intentions are very questionable by these big tech companies.  There has to be better way than giving out your phone number.
